src/models/FGAuxiliary.cpp

```diff
@@ -372,7 +384,7 @@ double FGAuxiliary::GetDistanceRelativePosition(void) const
 {
   auto ic = FDMExec->GetIC();
   return in.vLocation.GetDistanceTo(ic->GetLongitudeRadIC(),
-                                    ic->GetGeodLatitudeRadIC())* fttom;
+                                    ic->GetGeodLatitudeRadIC());
```

Comment:
Same as above. This method is used to compute the property `position/distance-from-start-mag-mt` which is expressed in meters.
https://github.com/JSBSim-Team/jsbsim/blob/83e3df2c49c13eae78ab7853848cdd9e02c6c5ac/src/models/FGAuxiliary.cpp#L448